src/Intercom.Tests/Clients/ContactClientTest.cs

```diff
@@ -24,31 +24,27 @@ public ContactClientTest()
         }
 
         [Test()]
-        [ExpectedException(typeof(ArgumentNullException))]
         public void Create_WithNull_ThrowException()
         {
-            contactsClient.Create(null);
+            Assert.Throws<ArgumentNullException>(() => contactsClient.Create(null));
         }
 
         [Test()]
-        [ExpectedException(typeof(ArgumentException))]
         public void Update_NoIdOrUserIdOrEmail_ThrowException()
         {
-            contactsClient.Update(new Contact());
+            Assert.Throws<ArgumentException>(() => contactsClient.Update(new Contact()));
         }
 
         [Test()]
-        [ExpectedException(typeof(ArgumentNullException))]
         public void ListByEmail_NoEmail_ThrowException()
         {
-            contactsClient.List(String.Empty);
+            Assert.Throws<ArgumentNullException>(() => contactsClient.List(String.Empty));
         }
 
         [Test()]
-        [ExpectedException(typeof(ArgumentException))]
         public void Delete_NoIdOrUserIdOrEmail_ThrowException()
         {
-            contactsClient.Delete(new Contact());
+            Assert.Throws<ArgumentNullException>(() => contactsClient.Delete(new Contact()));
         }
     }
 }
```

Comment:
re: my earlier comment on company tests ... I think you have the right approach alright, just make the minimum changes we need for this migration and then we can think about improving the unit tests i a more systematic way 👍

Reply:
Yep that was my train of thought as well. Making this as minimal as possible, and then add to the roadmap to redo the tests. 👍